README.md

```diff
@@ -38,7 +38,40 @@ at `/config` inside the container. It will be copied into place prior to buildin
 Note that `make olddefconfig` will be run regardless to ensure the latest
 options have been applied.
 
-## Reproducibile builds
+## Building kernels in Qubes
+
+Here's how to set up a build environment in [Qubes], suitable for use with [SecureDrop].
+The build requires `docker`, so make sure your TemplateVM has docker configured.
+
+```
+qvm-create sd-kernel-builder --template debian-10 --label purple
+qvm-prefs sd-kernel-builder vcpus $(nproc)
+qvm-volume resize sd-kernel-builder:private 50G
+
+```
+
+Then add the following customization to the AppVM to ensure
+the private volume is used for the build:
+
+```
+$ cat /rw/config/qubes-bind-dirs.d/50_user.conf
```

Comment:
Enumerating instructions might be helpful here, like so:

1. Create a bind-dirs directory and a new file called '50_user.conf'
```
sudo mkdir /rw/config/qubes-bind-dirs.d
sudo vi /rw/config/qubes-bind-dirs.d/50_user.conf
```
2. Edit '50_user.conf' to append '/var/lib/docker' to the binds variable:
```
binds+=( '/var/lib/docker' )
```
3. Reboot the AppVM
```
sudo reboot
```